### ISSUE TITLE
Request for sample application

### ISSUE THREAD

**Issue by abeq** [nakamura-to/komapper#1]
Web-DB application such as PetShop of Doma.
I do not need PetShop, but I need sample for typical Web-DB application called as 'IREPON-DASHIPON'.

**Comment by nakamura-to** [comment]
I know 'IREPON-DASHIPON'. It means the application which has few business logics and mainly reads/writes data directly from/to DB, right? 😄 

Any Web-DB application depends on a specific web framework, but we'd like to independent from all web frameworks. For the present, we provide DB access examples only.

**Comment by abeq** [comment]
> we'd like to independent from all web frameworks.
I understand this idea, but most DB applications are as WEB-DB applications, so I think the sample implementation is important for popularization.
You can choose the web framework that you think is best to implement.

About IREPON-DASHIPON, your answer is right. Because it will be simple for sample.

**Comment by nakamura-to** [comment]
> the sample implementation is important for popularization.

I agree with you. However, it's too hard to maintain the web sample application for us.
The reasons are as follows:
- most parts of the application are web-related, in other words, DB-related parts are very tiny
- the libraries which the web framework depends on are out of date soon
- we are short of hands

We are glad that you would contribute to it.
